### PR TITLE
update CHANGELOG and README

### PR DESCRIPTION
**Pull request by bdworth**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -27,6 +24,9 @@ buildscript {
     }
```
Comment (uschi2000) [inline README.md]: can you add block here please:

````
buildscript {
  repositories {
    jcenter()
  }
  dependencies {
  ...
  }
}
````

File: README.md [hunk 2]
```diff
@@ -27,6 +24,9 @@ buildscript {
...
 }
```
Comment (uschi2000) [inline README.md]: add a block here as well:

````
  repositories {
    jcenter()
  }
````

File: README.md [hunk 3]
```diff
@@ -34,6 +34,8 @@ apply plugin: 'com.palantir.baseline-findbugs'
 apply plugin: 'com.palantir.baseline-idea'
 ```
 
+- Run ``./gradlew baselineUpdateConfig`` to download the latest config files
```
Comment (uschi2000) [inline README.md]: remove "latest"; the task installs the one corresponding to the plugin version

**Comment by bdworth** [comment]
moved to https://github.com/palantir/gradle-baseline/pull/16

**Pull request closed without merging**